src/backend/expungeservice/models/case.py

```diff
@@ -69,4 +69,4 @@ def _ignore_open_case(self):
         return 'violation' in self.violation_type.lower() or 'municipal parking' == self.violation_type.lower()
 
     def _closed(self):
-        return self.current_status == 'Closed' or self.current_status == 'Inactive' or self.current_status == 'Purgable'
+        return self.current_status == 'Closed' or self.current_status == 'Inactive' or self.current_status == 'Purgable' or self.current_status == 'Bankruptcy Pending'
```

Comment:
It might be good to extract these keywords into a tuple. And then we can do `self.current_status in CLOSED_STATUS`